modules/registry-configuring-storage-aws-upi.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * architecture/registry
+//* architecture/installation-.adoc
+
+[id="registry-configuring-storage-aws-upi-{context}"]
+= Configuring registry storage for AWS with UPI
+
+During installation, you must configure your registry to use storage.
+
+.Prerequisites
+
+* A cluster on AWS with upi
+
+.Procedure
+
+. Check to see if your `image-registry-private-configuration` secret exists:
```

Comment:
`image-registry-private-configuration` shouldn't be modified by users. After today's scrum I'd say there are 2 options:

  * the cloud credentials are sufficient to create an S3 bucket, in this case the operator will do the storage configuration automatically;
  * the operator are not capable of creating S3 buckets, in this case the administrator should create an S3 bucket, setup [a Bucket Lifecycle Policy][1] to abort incomplete multipart uploads that are one day old, and fill the storage configuration in configs.imageregistry.operator.openshift.io/cluster.

[1]: https://docs.aws.amazon.com/AmazonS3/latest/dev/mpuoverview.html#mpu-abort-incomplete-mpu-lifecycle-config